pkg/source/source.go

```diff
@@ -98,7 +99,7 @@ func (ks *Kind) Start(handler handler.EventHandler, queue workqueue.RateLimiting
 	}
 
 	// Lookup the Informer from the Cache and add an EventHandler which populates the Queue
-	i, err := ks.cache.GetInformer(ks.Type)
+	i, err := ks.cache.GetInformer(context.TODO(), ks.Type)
```

Comment:
Do we need to add `context.Context` to the `Source` interface's `Start` method to make this cancellable?

Reply:
I guess, but maybe we can do that in a follow-up?